### PR TITLE
Documentation: using custom connectors

### PR DESCRIPTION
**Pull request by alafanechere**
## What
Add documentation about how to use custom connectors and a private docker registry in the Operator Guides section.
This is a question often received by the USE team on slack.

## How
Explain how to:
* Create a private Docker registry
* Authenticate Airbyte to the Docker registry
* Add a custom connector in the UI

### CODE REVIEW

File: docs/SUMMARY.md [hunk 1]
```diff
@@ -31,6 +31,7 @@
     * [Transformations with Airbyte (Part 3/3)](operator-guides/transformation-and-normalization/transformations-with-airbyte.md)
   * [Configuring Airbyte](operator-guides/configuring-airbyte.md)
     * [Sentry Integration](operator-guides/sentry-integration.md)
+  * [Using custom connectors](operator-guides/using-custom-connectors.md)
```
Comment (marcosmarxm) [inline docs/SUMMARY.md:34]:
```suggestion
  * [Using Custom Connectors](operator-guides/using-custom-connectors.md)
```

File: docs/operator-guides/using-custom-connectors.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+| Google Cloud   | Artifact Registry           | [Quickstart](https://cloud.google.com/artifact-registry/docs/docker/quickstart)|
+| AWS            | Amazon ECR                  | [Getting started with Amazon ECR](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html)|
+| Azure          | Container Registry          | [Quickstart](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-get-started-portal#:~:text=Azure%20Container%20Registry%20is%20a,container%20images%20and%20related%20artifacts.&text=Then%2C%20use%20Docker%20commands%20to,the%20image%20from%20your%20registry.)|
+| DockerHub      | Repositories                | [DockerHub Quickstart](https://docs.docker.com/docker-hub/)|
+| Self hosted    | Open-source Docker Registry | [Deploy a registry server](https://docs.docker.com/registry/deploying/)|
+
+## 2. Authenticate to your private Docker registry
+To push and pull images to your private Docker registry, you need to authenticate to it:
```
Comment (marcosmarxm) [inline docs/operator-guides/using-custom-connectors.md:28]: Maybe use Private Container Registry?

**Review by marcosmarxm — APPROVED** [review]
small comment but looks terrific!

File: docs/operator-guides/using-custom-connectors.md (hunk 2 above)
Comment (alafanechere) [reply docs/operator-guides/using-custom-connectors.md:28]: I think the common naming is _Docker registry,_ and in our case we want it to be private 😄 . _Container registry_  is a cloud service name.

**Pull request merged**